README.md

```diff
@@ -28,6 +28,23 @@ if they are valid then the cookie is updated to indicate this otherwise an error
 * if there is a cookie but it indicated the the authentication is expired then the user is sent off to Google to renew their session.
 On returning from Google the existing cookie is updated with the new expiry time.
 
+## Authorising to the s3 bucket
+If the application is running in the same AWS account as the pan-domain bucket (Workflow) then apps can access this by granting read
+access to the pan-domain bucket in the cloudformation script.
+For applications outside the Workflow account there is an example sts:AssumeRole script. You need to add your AWS accountId in the 
+parameters and upload this to the Workflow account and get the roleArn.
+
+In your PanDomainAuthActions include this:
+
+```
+override def awsCredentialsProvider  = {
+  new AWSCredentialsProviderChain(new STSAssumeRoleSessionCredentialsProvider({{roleArn}}, "yourapp"),
+                                  new ProfileCredentialsProvider("workflow"))
```

Comment:
Just saw the note below. Would be nice to have a way that doesn't require ad-hoc credentials for the workflow account.